cudamapper/src/overlapper.cpp

```diff
@@ -306,7 +306,8 @@ void Overlapper::rescue_overlap_ends(std::vector<Overlap>& overlaps,
                                      const std::int32_t extension,
                                      const float required_similarity)
 {
-
+    (void)extension;
```

Comment:
Eric should check why these values are not being used. I'll ping him directly as I cannot add him to this PR

Reply:
I can't seem to add comments to the PR, but these values should be used in the call to `details::overlapper::extend_overlap_by_sequence_similarity(overlap, query_view, target_view, 100, 0.9)`. I'm not sure why they are hard-coded as extension=100 and required_similarity=0.9 but it's quite possibly my mistake.